packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -130,29 +130,6 @@ describe("TableComponent", () => {
 		component = hostComponent.tableComponent;
 	});
 
-	describe("on initialization", () => {
-		it("should set internal component properties", () => {
-			expect(hostFixture).toBeDefined();
-			expect(component).toBeDefined();
-
-			expect(component.dialogService).not.toBeNull();
-			expect(component.dialogService).toBeDefined();
-			expect(component.logger).not.toBeNull();
-			expect(component.logger).toBeDefined();
-		});
-
-		it("should NOT have any inputs set", () => {
```

Comment:
I think we should keep this test to be sure what the initial value of the properties is.

Reply:
I came across alot of these tests, but I don't really understand them. They don't test anything functional right?

Reply:
Not functional but more internal stuff... otherwise it will be hard to track down issues due to a wrong initialization of our properties...